Dockerfile

```diff
@@ -1,20 +1,25 @@
-FROM amsterdam/python
+FROM amsterdam/python:3.8-buster AS builder
 MAINTAINER datapunt@amsterdam.nl
 
-ENV PYTHONUNBUFFERED 1
+COPY requirements* ./
+ARG PIP_REQUIREMENTS=requirements.txt
+RUN pip install --no-cache-dir -r $PIP_REQUIREMENTS
 
-EXPOSE 8000
+# Start runtime image
+FROM amsterdam/python:3.8-slim-buster
 
-RUN rm -rf /data
-RUN mkdir /data && chown datapunt /data
+# Copy python build artifacts from builder image
+COPY --from=builder /usr/local/bin/ /usr/local/bin/
+COPY --from=builder /usr/local/lib/python3.8/site-packages/ /usr/local/lib/python3.8/site-packages/
 
-WORKDIR /app/
-COPY requirements.txt /app/
-RUN pip install -r requirements.txt
+WORKDIR /app
+RUN rm -rf /data \
```

Comment:
the `rm -rf` looks kind of fishy. If the directory does exist, this is masking a problem that may resurface later.

Reply:
Agreed. I copied it from the old `Dockerfile`, but have addresses this issue already using an `.dockerignore` file